### PR TITLE
Improve translations for Brazilian portuguese

### PR DESCRIPTION
**Pull request by wenderjean**
Improving translations for Brazilian portuguese.

### CODE REVIEW

**Comment by davidsilveira** [comment]
Seems good to me!

**Comment by glebm** [comment]
Can you please run:

```shell
bundle exec i18n-tasks normalize
```

File: config/locales/pt-BR.yml [hunk 1]
```diff
@@ -257,17 +257,17 @@ pt-BR:
...
       recent_activity: :thredded.recent_activity
       send_private_message: Enviar mensagem privada
       started_topic_html: Começou %{topic_link}
       started_topics_count:
         one: Começou um tópico
         other: tópicos iniciados %{count}
       user_posted_in_topic_html: "%{user_link} postou em %{topic_link}"
-      user_since_html: Utilizador desde %{time_ago}
+      user_since_html: Usuário desde %{time_ago}
       user_started_topic_html: "%{user_link} começou %{topic_link}"
```
[the remark this answers is not in the document]
Comment (glebm) [reply config/locales/pt-BR.yml]: Up to you! The English version is closer to `"%{user_link} começou o %{topic_link}"`

**Comment by glebm** [comment]
Thank you!

**Pull request merged**